### PR TITLE
Fixed issue with Decimal String conversion resulting in more than 16 …

### PR DESCRIPTION
**Pull request by rvantklooster**
Fixed issue with Decimal String conversion resulting in more than 16 characters and added unit test.
See https://github.com/dcmjs-org/dcmjs/issues/366

### CODE REVIEW

**Review by pieper — APPROVED** [review]
Makes sense to me.

@swederik you thumbed up the issue, can you take a look at this before we merge just for another set of eyes?  Do you think there should be a comment explaining what the code does?

**Comment by swederik** [comment]
Yes completely agree, the code is good but a short comment about why it exists will be helpful for the next person who reads it. Especially happy that @rvantklooster added a test! Thanks!

**Review by swederik — APPROVED** [review]
Looks good but please add a short comment above the logic explaining why it is there, thanks!

**Comment by rvantklooster** [comment]
I updated the documentation (in line with the documentation of its pydicom counterpart). Nice to see this quick turnaround!

**Pull request merged**
